tests/integration/targets/test_provision_and_deprovision/playbooks/test_provision_and_deprovision.yml

```diff
@@ -124,6 +124,13 @@
                     - " IF LASTCC = 0 THEN DO"
                     - "   DELETE {{ DFH_REGION_DATA_SETS | upper }}.DFHDMPB"
                     - " END"
+          ignore_errors: true
+          register: cleanup_result
+
+        - name: Check cleanup
+          ansible.builtin.assert:
+            that:
+              - cleanup_result.ret_code.code <= 4
```

Comment:
Checking for 0 and 4 specifically here might be better

Reply:
Fixed